phys/module_surface_driver.F

```diff
@@ -1412,6 +1413,9 @@ SUBROUTINE surface_driver(                                         &
                                                                                tkmg3d,         &
                                                                                tkdry3d,        &
                                                                                tksatu3d
+    INTEGER , OPTIONAL , INTENT(IN)                            :: shalwater_rough 
+    REAL,     OPTIONAL , INTENT(IN)                            :: shalwater_depth 
+    real, OPTIONAL , dimension(ims:ime,jms:jme ),intent(inout) :: water_depth 
```

Comment:
@phawbeck The new variables here do not need to be declared as optional variables.